lib/vtools_fetcher.py

```diff
@@ -1,132 +1,15 @@
+load_dotenv()
 
-        self.description += f"\n---" \
-            f"\nEvent ID: {self.id}" \
-            f"\nHost: {self.host}" \
-            f"\nHost Email: {self.host_email}" \
-            f"\nLocation: {self.location}" \
-            f"\nStart Date: {self.start_date.strftime(self.VTOOLS_DATETIME_FORMAT)}" \
-            f"\nEnd Date: {self.end_date.strftime(self.VTOOLS_DATETIME_FORMAT)}" \
-            f"\nURL: {self.url}" \
-            f"\n---"
-
-    def __str__(self):
-        return f"{self.title} | {self.host} | {self.start_date} - {self.end_date} | {self.location}"
-    
-    def __repr__(self):
-        return str(self)
-    
-    def __eq__(self, value):
-        return self.id == value.id and \
-            self.title == value.title and \
-            self.start_date == value.start_date and \
-            self.end_date == value.end_date and \
-            self.location == value.location and \
-            self.host == value.host and \
-            self.host_email == value.host_email and \
-            self.url == value.url and \
-            self.description == value.description
-
-    def __ne__(self, value):
-        return not self.__eq__(value)
-    
-    
-    def to_calendar_event(self):
-        return {
-            "organizer": {
-                "email": self.host_email,
-                "displayName": self.host,
-            },
-            "status": "confirmed",
-            "id": self.id,
-            "summary": self.title,
-            "location": self.location,
-            "description": self.description,
-            "start": {
-                "dateTime": self.start_date.isoformat(),
-                "timeZone": "America/Bogota",
-            },
-            "end": {
-                "dateTime": self.end_date.isoformat(),
-                "timeZone": "America/Bogota",
-            },
-        }
-    
-    @staticmethod
-    def from_calendar_event(event):
-        metadata = event['description'].split('\n\n---')[1]
-        host = metadata.split('\nHost: ')[1].split('\n')[0]
-        host_email = metadata.split('\nHost Email: ')[1].split('\n')[0]
-        url = metadata.split('\nURL: ')[1].split('\n')[0]
-
-        return Event(
-            id=str(event['id']),
-            title=event['summary'],
-            cal_status=event['status'],
-            start_date=datetime.strptime(event['start']['dateTime'], Event.CALENDAR_DATETIME_FORMAT).replace(tzinfo=None),
-            end_date=datetime.strptime(event['end']['dateTime'], Event.CALENDAR_DATETIME_FORMAT).replace(tzinfo=None),
-            location=event['location'],
-            host=host,
-            host_email=host_email,
-            url=url,
-            description=event['description'].split('\n\nMore info at ')[0],
-        )
+BASE_URL = os.getenv('VTOOLS_API')
+EVENT_LIMIT = 2000
 
 def filter_event_by_country(event, country_id):
     country_data = event['relationships']['country'].get('data')
+    print(country_data)
```

Comment:
My suggestion is removing this print, because there is not a standard logging format inside `lib/`. I opened #2 
```suggestion
```